Minor tweaks and edits (#59)
bufbuild/docs.buf.build: docs/generate/usage.md

## Run generate

You can run `buf generate` on your module by specifying the filepath to
the directory containing the `buf.yaml`. In the above example, you can target

> the `buf.build/acme/petapis` input defined in the current directory like so:

Author: There is no reference to petapis in this file or example. I wasn't really sure what to do about 

> module by specifying the filepath to the directory containing the `buf.yaml`.

because as far as I understand generate is always run in the directory with the yaml, unless using workspaces.

Reviewer: We refer to `buf.build/acme/weather` in plenty of other files though. It's simply included to be an example for a module that can be used as input.

I agree that this information doesn't need to be here, but we should add the module reference (with `buf.build/acme/petapis`) to the `Common use cases` below.

> because as far as I understand generate is always run in the directory with the yaml, unless using workspaces.

You can run `buf generate` on an input that doesn't define a `buf.yaml`. Just like `buf build`, `buf` will assume that the directory is a source that defines `.proto` sources relative to that root.

Author: I think I was confused as the two examples in text (subdir and module) are removed from the example, which specifies nothing which defaults to the working directory. Updated and also added to the "common use cases" below, lmk if this makes sense.

## Run generate

> To generate for a buf module in your current directory, simply run:

Reviewer: Use the word `input` so that it's clear it doesn't have to be a module (like elsewhere in the docs):
```suggestion
To generate for the input in your current directory, simply run:
```

```sh
$ buf generate
```

> You can also run `buf generate` on a module by specifying the filepath to the
> directory containing the proto root. For example if your protos are in directory
> `foo`:

Reviewer: I'd make this a bit more explicit.

```suggestion
You can also run `buf generate` on an input by specifying the filepath to the
directory containing the root of your `.proto` definitions. For example if all of
your `.proto` files are in directory `foo`:
```

# Uses the current directory as input, and assumes a `buf.gen.yaml` also exists in the current directory.
$ buf generate

# Uses the buf.build/acme/petapis module as input, using a `buf.gen.yaml` in the current directory.

> $ buf generate buf.build/acme/petapis

Reviewer: Nice 👏
